### PR TITLE
Changed analytics data structure in YuktaMedia Analytic Adapter

### PR DESCRIPTION
**Pull request by shrikantpatwari**
## Type of change

- [x] Feature
- [x] Refactoring (no functional changes, no api changes)
## Description of change

Changed analytics data structure in YuktaMedia Analytic Adapter
 

1. Removed use of deprecated functions from util library 
2. Made changes for sizes which moved inside mediaTypes.banner

- contact email of the adapter’s maintainer
- [x] official adapter submission
[shrikantpatwari@gmail.com](mailto:shrikantpatwari@gmail.com)
[info@yuktamedia.com](mailto:info@yuktamedia.com)

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/ for name update is below

https://github.com/prebid/prebid.github.io/pull/1714

### CODE REVIEW

**Comment by shrikantpatwari** [comment]
Hello @mkendall07 

We have made changes in our Analytics Adaptor and created PR for same but on circle ci some of e2e tests are failing. Hence we took original master branch pull and made complete build to verify all tests are passing, however we found that exactly same e2e test are failing on master branch in main repo as well. So we assume there is no code changes we have made which breaking the build. So request you to please look into it and merge our PR.

Thanks

File: modules/yuktamediaAnalyticsAdapter.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+function send(data, status) {
+  let location = utils.getWindowLocation();
+  let secure = location.protocol == 'https:';
+  if (typeof data !== 'undefined' && typeof data.auctionInit !== 'undefined') {
+    data.auctionInit = Object.assign({ host: location.host, path: location.pathname, search: location.search }, data.auctionInit);
+  }
+  data.initOptions = initOptions;
+
+  let yuktamediaAnalyticsRequestUrl = url.format({
+    protocol: secure ? 'https' : 'http',
```
Comment (jsnellbaker) [inline modules/yuktamediaAnalyticsAdapter.js]: As part of Prebid 3.x - all requests need to be made with `https`.  Can you update this logic here to hard-code the protocol value?

**Comment by jsnellbaker** [comment]
To note as well - I ran the browserstack/circleci tests manually and they passed.  So you should be fine on that front.

File: modules/yuktamediaAnalyticsAdapter.js (hunk 1 above)
Comment (shrikantpatwari) [reply modules/yuktamediaAnalyticsAdapter.js]: Hi @jsnellbaker 

I have update protocol to https as per requirement and your suggestion. Can you please re-review and merge into master.

Thanks.

**Pull request merged**